catalog/app/components/PaymentDialog/index.js

```diff
@@ -112,7 +112,7 @@ class PaymentDialog extends React.PureComponent { // eslint-disable-line react/p
             />
             <br />
             <br />
-            <Pricing includeSignIn={false} />
+            <Pricing takeAction={false} />
```

Comment:
there is no `includeSignIn` prop on the `Pricing` component, so I've introduced `takeAction` prop (`true` by default) to be able to completely hide the `<TakeAction />` there